xsuite/src/world/lsworld.test.ts

```diff
@@ -625,6 +654,76 @@ test.concurrent("LSWorld.callContract", async () => {
   });
 });
 
+test.concurrent("LSWorld.addKvs", async () => {
+  using world = await LSWorld.start();
+  const wallet = await world.createWallet({
+    balance: 10n ** 18n,
+    kvs: { "01": "02", "03": "04" },
+  });
+  await world.addKvs(wallet, {
+    "01": "05",
+    "06": "07",
+  });
+  assertAccount(await wallet.getAccount(), {
+    balance: 10n ** 18n,
+    kvs: { "01": "05", "03": "04", "06": "07" },
+  });
+});
+
+test.concurrent("LSWorld.addEsdts", async () => {
+  using world = await LSWorld.start();
+  const wallet = await world.createWallet({
+    balance: 10n ** 18n,
+    kvs: {
+      esdts: [
+        { id: "01", amount: 10n ** 18n },
+        { id: "02", amount: 10n ** 18n },
+      ],
+    },
+  });
+  await world.addEsdts(wallet, [
+    { id: "01", amount: 20n ** 18n },
+    { id: "03", amount: 10n ** 18n },
+  ]);
+  assertAccount(await wallet.getAccount(), {
+    balance: 10n ** 18n,
+    kvs: {
+      esdts: [
+        { id: "01", amount: 20n ** 18n },
+        { id: "02", amount: 10n ** 18n },
+        { id: "03", amount: 10n ** 18n },
+      ],
+    },
+  });
+});
+
+test.concurrent("LSWorld.addMappers", async () => {
+  using world = await LSWorld.start();
+  const wallet = await world.createWallet({
+    balance: 10n ** 18n,
+    kvs: {
+      mappers: [
+        { key: "01", value: e.Buffer("02") },
```

Comment:
For the key, let's put a name, e.g. "key1"

For the value, let's use bigint, e.g. `e.U(2)`.

Same for the other ones.

Reply:
Applies to all the other tests of the kind.